community/contribute/README.md

```diff
@@ -15,3 +15,15 @@ To submit an RFP, head to the [RFP forum category](https://forum.pokt.network/c/
 A PEP is a type of proposal that requests a POKT grant from the DAO treasury. You could get a POKT grant for contributions to the Pocket Network ecosystem.
 
 Read more about [submitting a proposal](../governance/proposals.md).
+
+### Submit a PR - Become an Active Pocket V1 Contributor
+
+You can become an open-source contributor by submitting code contributions to the [core Pocket repository](https://github.com/pokt-network/pocket) or the [specifications](https://github.com/pokt-network/pocket-network-protocol).
+
+All the issues are documented and tagged on [Github](https://github.com/pokt-network/pocket/issues), and the [starter task](https://github.com/pokt-network/pocket/issues?q=is%3Aissue+is%3Aopen+label%3A%22starter+task%22) tag can be used to filter for simpler issues to get you acquainted with the codebase.
+
+All of these contributions are free and under the MIT license, but may be partially rewarded by the [scholarship fund](community/contribute/scholarships.md). Larger contributions from repeat contributors will be evaluated by the DAO through separate proposals.
+
+In addition to contributing to the future of Web3, you will also get mentorship and experience by working with the core Pocket team, as well as present and discuss your work in our weekly contributor hours.
+
+Check out the [v1-research](https://discord.com/channels/553741558869131266/936666517498650644) and [v1-dev](https://discord.com/channels/553741558869131266/986789914379186226) channels in discord to connect with the team.
```

Comment:
Suggest adding hashtags to these links to make it more obvious that they are channels. Also, please capitalize Discord.

Reply:
Done.